docs/source/installation/preparing-to-migrate.rst

```diff
@@ -0,0 +1,23 @@
+Preparing to Migrate
+====================
+
+If you are migrating to your new environment from an existing
+environment, it is important that mobile users experience as little
+disruption as possible.
+
+You can minimize disruption by preparing the following in advance:
+
+#. Register a DNS alias that initially points to the existing
```

Comment:
Nit: maybe add here "points to the existing environment on commcarehq.org" to make it obvious what the initial environment is.

Reply:
b7d868c72